Sources/eppo/RuleEvaluator.swift

```diff
@@ -30,16 +30,49 @@ struct FlagEvaluation {
         subjectAttributes: SubjectAttributes,
         allocationKey: Optional<String>,
         variation: Optional<UFC_Variation>,
-        variationType: [UFC_VariationType],
+        variationType: Optional<UFC_VariationType>,
         extraLogging: [String: String],
-        doLog: Bool
+        doLog: Bool,
+        isConfigObfuscated: Bool
     ) -> FlagEvaluation {
+        // If the config is obfuscated, we need to unobfuscate the allocation key.
+        var allocationKeyFinal: String = allocationKey ?? ""
```

Comment:
minor but isntead of "final" how about "decoded" seems more explicit?

Will also help us prevent variable names like `allocationKeyFinalFinalV3` as our code grows 😜